src/api_client/mod.rs

```diff
@@ -173,7 +173,7 @@ impl racal::reqwest::ApiClient<NoAuthentication> for UnauthenticatedCVR {
 		&self, req: RequestBuilder,
 	) -> Result<RequestBuilder, racal::reqwest::ApiError> {
 		self.http_rate_limiter.until_ready().await;
-		Ok(req)
+		Ok(req.header("Content-Type", "application/json"))
```

Comment:
yeah I've looked into this a bit more and I'd say it should be included in the `racal` package anyway since the `body` function inside of `Queryable` returns an `Option` with a `serde_json::Result` and I think it's safe to assume that if something is there then it's json data.
but correct me if anything I've said here is wrong since I'm not fully confident about this

I also found that `neos_rs` had the same header issue, so fixing it only in this package wouldn't actually fix everything